kDrive/UI/Controller/Files/Preview/PreviewViewController.swift

```diff
@@ -449,27 +449,9 @@ final class PreviewViewController: UIViewController, PreviewContentCellDelegate,
         }
 
         let file = previewFiles[index]
-        let previewError = OfficePreviewError(fileId: fileId)
         if file.convertedType == .spreadsheet || file.convertedType == .presentation || file.convertedType == .text {
```

Comment:
I think we have a bigger naming issue because our `OnlyOfficePreviewCell` can also preview other documents that are not necessarily handled by OnlyOffice.
To check for OnlyOffice compatibility we have a `.supportedBy -> onlyOffice`

For now, I added a document `ConvertedType.documentTypes` property that's more inline with the project.

Reply:
Cool I like the set check. Thanks.